redash/query_runner/python.py

```diff
@@ -109,6 +109,12 @@ def custom_write(obj):
         """
         return obj
 
+    @staticmethod
+    def custom_inplacevar(op, x, y):
+        globs = {'x': x, 'y': y}
+        exec 'x'+op+'y' in globs
+        return globs['x']
```

Comment:
Are we sure this implementation is safe? Maybe have a whitelist of allowed `op` values?

Also, doesn't RestrictedPython have its own default implementation for this?

Reply:
Maybe we should upgrade to 4.0?

Reply:
Sorry, I'm not sure. RestrictedPython 4.0(stable version) hasn't released yet.

Reply:
Got it. If we don't upgrade to RP 4.0, then let's use their list of safe arguments. Thanks!